### DISCUSSION TITLE
Capturing API Calls without Full TracingInstrumentation

### DISCUSSION THREAD

**Discussion by stevenG-dv** [grafana/faro-web-sdk#223] [Q&A] [answered]
Documentation says that using TracingInstrumentation is not to be used lightly as it is extremely taxing. However, I do want to still capture: (console, fetch, xhr, and uncaught exceptions) without also capturing every single other thing that happens in the application. 
Is this possible (without utilizing any react related implementations)? -- This is for an Angular app.

**Comment by codecapitano** [comment, marked as answer]
Hi @stevenG-dv 

We are currently working on specific instrumentations for Fetch and XmlHttpRequest.
Additionally we will add traceIDs' without the need to use the TraceInstrumentation.

There is a[ wip implementation of the FetchInstrumentation](https://github.com/grafana/faro-web-sdk/tree/main/experimental/instrumentation-fetch). Currently provided as an experimental package.

Cheers,
Marco

/cc @cedricziel
